config/environments/production.rb

```diff
@@ -31,35 +30,43 @@
   # config.action_dispatch.x_sendfile_header = "X-Sendfile" # for Apache
   # config.action_dispatch.x_sendfile_header = "X-Accel-Redirect" # for NGINX
 
-  # Include generic and useful information about system operation, but avoid logging too much
-  # information to avoid inadvertent exposure of personally identifiable information (PII).
-  config.log_level = :info
+  # Assume all access to the app is happening through a SSL-terminating reverse proxy.
+  # Can be used together with config.force_ssl for Strict-Transport-Security and secure cookies.
+  # config.assume_ssl = true
+
+  # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
+  config.force_ssl = true
```

Comment:
@alexdesi This is the same as frontend? If you're using this option, you'll need to enable `assume_ssl` I think ? 

@willfish @amberstarlight I'm not familiar on why this wasn't working before but it does appear to be working without `assume_ssl`????

Reply:
CDN and load balancer terminate SSL and reverse proxy onto the apps, yeah.

The deployment will fail with force_ssl = true and we should instead be using assume_ssl = true

https://github.com/trade-tariff/trade-tariff-platform-aws-terraform/blob/main/environments/common/application-load-balancer/alb.tf#L26-L36

The load balancer checks the health of the apps via these endpoints over http:

```terraform
locals {
  services = {
    admin = {
      host             = ["admin.*"]
      healthcheck_path = "/healthcheckz"
    }

    signon = {
      host             = ["signon.*"]
      healthcheck_path = "/healthcheck/live"
    }

    backend_uk = {
      paths            = ["/uk/api/beta/*"]
      healthcheck_path = "/healthcheckz"
    }

    backend_xi = {
      paths            = ["/xi/api/beta/*"]
      healthcheck_path = "/healthcheckz"
    }

    duty_calculator = {
      paths            = ["/duty-calculator/*"]
      healthcheck_path = "/healthcheckz"
    }

    search_query_parser = {
      paths            = ["/api/search/*"]
      healthcheck_path = "/healthcheckz"
    }

    frontend = {
      paths            = ["/*"]
      healthcheck_path = "/healthcheckz"
    }
  }
}
```

Reply:
https://github.com/rails/rails/pull/47139/commits/17ee01f22b1a59987c47e78625b8f8fd9513a740

Reply:
I've also set
`config.assume_ssl = true`
for the TT Admin